src/plone/restapi/serializer/relationfield.py

```diff
@@ -17,7 +18,8 @@
 @adapter(IRelationValue)
 @implementer(IJsonCompatible)
 def relationvalue_converter(value):
-    if value.to_object:
+    mtool = api.portal.get_tool("portal_membership")
+    if value.to_object and mtool.checkPermission("View", value.to_object):
```

Comment:
this is too much overhead and probably slow. Use the Zope basics, those are perfectly fine here:
```
from AccessControl.SecurityManagement import getSecurityManager
getSecurityManager().checkPermission(permission, obj)
```

Reply:
btw, if you anyway use `plone.api`, there is `api.user.check_permission` too as an alternative.

Reply:
If I am not wrong, plone.api may only be used in plone.restapi tests right? @plone/restapi-team

Reply:
I am not sure what the rules are, but if there are those rules, they must be documented somewhere.